pbxplore/structure/structure.py

```diff
@@ -96,7 +96,7 @@ def read_from_PDBx(cls, line, fields):
         http://mmcif.wwpdb.org/docs/tutorials/content/atomic-description.html
         """
         try:
-            dic = dict(zip(fields, line.split()))
+            dic = dict(list(zip(fields, line.split())))
```

Comment:
No need to make this a list explicitly. `dict` takes an iterable as its argument.